lib/tapioca/compilers/dsl/protobuf.rb

```diff
@@ -153,7 +166,7 @@ def field_of(descriptor)
               Field.new(
                 name: descriptor.name,
                 type: type,
-                init_type: "T.any(#{type}, T::Hash[#{key_type}, #{value_type}])",
+                init_type: "T.nilable(T.any(#{type}, T::Hash[#{key_type}, #{value_type}]))",
```

Comment:
These `init_type` changes also make sense, but seem like distinct from the change this PR is trying to make. Could we make these changes in a separate PR please?